### PR TITLE
Update jobs

### PR DESCRIPTION
**Pull request by SusanDefacto**
- add marketing job
- update Defacto text
- add Defacto text to consultant job

### CODE REVIEW

File: source/jobs/nl/_over-ons.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
+Defacto beweegt zich al jaren als toonaangevend ontwikkelaar van innovatieve software in het werkveld van Opleiden en Ontwikkelen. Defacto is uitgegroeid tot een solide, internationaal opererend, bedrijf. De kernactiviteiten bestaan uit het digitaliseren en automatiseren van opleidingsafdelingen, bedrijfsacademies en opleidingsinstituten.
 
 Defacto kent een platte organisatie en hoge mate van transparantie. De verantwoordelijkheid ligt op de werkvloer bij de mensen zelf en wordt door de mensen zelf ook bewaakt.
 
 We geloven dat het belangrijk is om in een leuke omgeving te kunnen werken. Onze werktijden zijn flexibel, het gaat om de kwaliteit die je levert.
-
-Je hebt de mogelijkheid gebruik te maken van een Bring Your Own Device-regeling, Defacto betaalt de helft van je device en hij is van jou.
-
-Elke vrijdag is er ruimte om je te verdiepen in nieuwe technieken en producten. Daarnaast krijg je jaarlijks een ruim individueel opleidingsbudget en ontvang je een variabele winstdeling.
```
Comment (fatboypunk) [inline source/jobs/nl/_over-ons.md]: So now we need to add this to the developer jobs?

Comment (RonaldDijkstra) [reply source/jobs/nl/_over-ons.md]: @SusanDefacto what do you think about this?

Comment (SusanDefacto) [reply source/jobs/nl/_over-ons.md]: This would probably the best. I will pick it up

Comment (RonaldDijkstra) [reply source/jobs/nl/_over-ons.md]: OK thanks

**Pull request merged**